mikrotik/provider.go

```diff
@@ -44,7 +62,10 @@ func mikrotikConfigure(d *schema.ResourceData) (c interface{}, err error) {
 	address := d.Get("host").(string)
 	username := d.Get("username").(string)
 	password := d.Get("password").(string)
-	c = client.NewClient(address, username, password)
+	tls := d.Get("tls").(bool)
+	ca_certificate := d.Get("ca_certificate").(string)
```

Comment:
ca_certificate => caCertificate to keep consistency with the code base's variable naming conventions.

Reply:
Done